.github/workflows/testAndPublish.yml

```diff
@@ -45,18 +47,22 @@ jobs:
     name: Set up matrix
     runs-on: ubuntu-latest
     outputs:
+      defaultRunner: ${{ steps.setMatrix.outputs.defaultRunner }}
```

Comment:
is this needed? why can't we just use `env.defaultRunner`? Does `runs-on` not allow using `env`?

Reply:
NO, at least according to visual studio  code, env is not available at this point.